codecov.yml

```diff
@@ -1,3 +1,3 @@
 # CHANGE THIS
 fixes:
-   - "/lowlydba/lowlydba/sqlserver/::"
+   - "/ansible_collections/lowlydba/sqlserver/::"
```

Comment:
I just checked the the one I have in `community.hashi_vault` and I don't have a leading slash, but it might not matter. If this was in the template it's probably fine, or perhaps more correct (I added mine before the template was updated). So anyway let's leave it with the slash for now unless it causes issues, we'll find out once we get some tests reporting.